src/main/java/tempest/Module.java

```diff
@@ -16,7 +16,7 @@
 public class Module {
     private UUID id;
     private String name;
-    private LinkedList<StudySession> studySessions = new LinkedList<StudySession>();
+    private LinkedList<StudySession> studySessions = new LinkedList<>();
```

Comment:
Why remove the type annotation?

Reply:
IntelliJ doesn't like them, says they are unnecessary: "Explicit type argument StudySession can be replace with <>"

Reply:
Okay, sweet.